### ISSUE TITLE
custom messages

### ISSUE THREAD

**Issue by yawboateng** [russellcardullo/terraform-provider-pingdom#78]
from pingdom UI, we can set a custom message when manually creating a check, i can seem to find info on how to do the same with this provider. is it possible?

**Comment by mateusz-fiolka** [comment]
Yeah, it would be good to have it. Without it there is no context regarding on which check failed in some situations.
At least that's what happens in our case (`pingdom` -> `pagerduty` -> `slack` integration)

**Comment by sankalp-khare** [comment]
The 3.1 API does support it (wasn't there in the previous API): https://docs.pingdom.com/api/#tag/Checks/paths/~1checks/post
